core/generalise.ml

```diff
@@ -223,6 +223,19 @@ let rigidify_quantifier : quantifier -> unit =
     | (_, _, `Row point)      -> rigidify_point point
     | (_, _, `Presence point) -> rigidify_point point
 
+let mono_type_args : type_arg -> unit =
```

Comment:
Please add a comment describing what happens here, since `mono_type_args` and the type signature aren't descriptive enough to see what's going on.

If I'm understanding correctly, the function is called to change all `Mono` type variable subkinds to `Any` after typechecking a function body is complete?